app/models/school.rb

```diff
@@ -144,6 +148,19 @@ def induction_tutor
     induction_coordinators.first
   end
 
+  def slug_candidates
+    [
+      :name,
+      %i[name urn],
+    ]
+  end
```

Comment:
So what can our urls look like? Are we sure school name is something we want in the url?

Reply:
two example urls:

/admin/schools/buckingham-primary-school
/admin/schools/buckingham-primary-school-110329

why would we not want school name in URL?

Reply:
I thought names are long, annoying, can have special characters in them. But if we are sure we want to have names in the url, then it's alright.

Reply:
the library we're using deals with special characters, but yeah, they can be pretty long